Basic-Car-Maintenance/Info.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this be at the top level? Or can it be defined inside the `Shared` folder?

If it can be in the `Shared` folder, can you put it right under the `GoogleService-Info.plist`
<img width="314" alt="Screenshot 2023-10-07 at 4 21 02 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/4ef90b4c-21cd-400d-b5c3-2714780baae6">